hw/ip/rv_dm/data/rv_dm_testplan.hjson

```diff
@@ -432,6 +432,30 @@
       stage: V1
       tests: ["rv_dm_dataaddr_rw_access"]
     }
+    {
+      name: halt_resume
+      desc: '''
+            Verify debug module halt/going/resume behavior.
+            - Program jtag_dmi_ral.dmcontrol.haltreq to 1'b1
+            - After a few usec, acknowledge halt by setting tl_mem_ral.halted to 0. (ack for hart index 0).
+            - Send a single valid command with following process:
+              - Set jtag_dtm_ral.dmi.op to dm::DTM_WRITE
+              - Set jtag_dtm_ral.dmi.address to dm::Command
+              - Set jtag_dtm_ral.dmi.data[31:24] to 0. (Command type AccessRegister).
+            - Check flags[0] to make sure internal state machine is at Go state by checking
```

Comment:
Should we check `jtag_dmi_ral.abstractcs.busy` is the appropriate value after each relevant write as well?

Reply:
Updated